### ISSUE TITLE
CyberPower CP1500PFCLCD (Unit 1) is malfunctioning

### ISSUE THREAD

**Issue by jdrch** [jdrch/Hardware#53]
[CyberPower CP1500PFCLCD](https://github.com/jdrch/Hardware/blob/master/UPS.md#cyberpower-cp1500pfclcd-unit-1) keeps turning off, and loses power entirely when a self-test is run.

* [CyberPower Battery Replacement Calculator](https://www.cyberpowersystems.com/battery-replacement-tool/)
* [CyberPower RB1280X2B Replacement Batteries](https://www.cyberpowersystems.com/product/ups/replacement-batteries/rb1280x2b/) (new connection block and cabling apparently included)
  * [Installation instructions](https://dl4jz3rbrsfum.cloudfront.net/documents/CyberPower_UM_RB1280X2B.pdf)
* [AJC 3rd Party Replacement Battery](https://batteryclerk.com/products/cyberpower-rb1280x2b-12v-9ah-ups-battery?variant=6848814022715) (need to order 2 and reuse existing connection block and cabling)
  * [Installation instructions](https://batteryclerk.com/pages/ups-battery-replacement)

**Comment by jdrch** [comment]
I installed the replacement battery last night and left the unit to charge. This morning, it reported fully charged batteries. However, it was unable to maintain power output during a self-test and also when I manually disconnected it from the outlet. This leads me to believe the unit itself may be dead.

Given that the other 2 CyberPower UPSes are 2 years younger than this unit and that this unit failed simultaneously with its battery, retaining the replacement batteries for the other units doesn't seem to make much sense.

[Battery Clerk's return policy](https://batteryclerk.com/pages/returns) involves the customer paying for shipping and a 15% restocking fee, so the best way forward may be to simply sell the replacements on Ebay.
